qa/tasks/cephfs/test_admin.py

```diff
@@ -19,28 +19,16 @@ class TestAdminCommands(CephFSTestCase):
     CLIENTS_REQUIRED = 1
     MDSS_REQUIRED = 1
 
-    def test_fsnames_can_only_by_goodchars(self):
-        n = 'test_fsnames_can_only_by_goodchars'
-        metapoolname, datapoolname = n+'-testmetapool', n+'-testdatapool'
-        badname = n+'badname@#'
+    def _check_pool_application_metadata_key_value(self, pool, app, key, value):
```

Comment:
Maybe we should hide the parent class so that unittest.loader doesn't try to run it.

```
del globals()['TestAdminCommands']
```

Reply:
Perhaps, yes. I'll try that out.

Reply:
Adding this change didn't have any impact. That's because vstart_runner.py loads test methods directly instead of classes, I think.

Reply:
@lxbsz Rishabh mentioned this to me and I checked. Even without your suggestion no tests were run for TestAdminCommands. What's the advantage of adding your suggestion?

Reply:
Why adding this won't have any impact possibly because for now in the parent class there has not `test_XXX` member function in it. More detail please see https://docs.python.org/3/library/unittest.html.

At least hide it could explicitly tell the unittest.loader do not try to run it and also tell someone don't manually run to test this class.

Reply:
@lxbsz @ajarr What about renaming it from TestAdminCommands to AdminCmdTestCase? Right now we don't have much in the class to  become anything like CephFSTestCase, but that might change in the future.

Reply:
> What about renaming it from TestAdminCommands to AdminCmdTestCase?

I've incorporated this change in latest update.